app/controllers/categories_controller.rb

```diff
@@ -299,6 +300,24 @@ def visible_groups
     render json: success_json.merge(groups: groups || [])
   end
 
+  def find
```

Comment:
It looks like we might have missed out on a test for this controller action?

Reply:
I added a test.